### PR TITLE
Documentation for the atomic components

### PR DESCRIPTION
**Pull request by MarkCorazon**
Its been discussed with Han to make this into a folder named docs, and made available into one file.

Please check this for any spelling or overall mistakes please. Its most usefull to click on files changed and search for the button on the right upper corner of the code comparison that says 'Display the rich diff' (bc we do be like that)

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+|-----------|---------|---------------------------------------|
+| property  | string  | Display's the current value that is pre-filled in the editor field.                               |
+
+## Boolean Component
+The boolean component takes a true/false value to determine if something is true or false in the form of a switch. When the user clicks the switch, it toggles it from false to true or vice versa.
+
+### How does it work in an ampersand script:
+To use the boolean component in your script, add the following statement:
+```
+REPRESENT projectActive TYPE BOOLEAN
```
Comment (hanjoosten) [inline docs/documentation.md:89]: In Ampersand, the name of Concept needs to start with an uppercase letter. Please fix this in all examples, so the reader isn't confused.

**Review by PatrickR079 — DISMISSED** [review]

File: docs/documentation.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+# Components
+With the ampersand project, its possible to create a variety of input fields and ways to display those input fields or simply data.
```
Comment (hanjoosten) [inline docs/documentation.md:2]: The name of the file `documentation.md` is very general. Please make it more specific to the content you are describing. maybe something like `interface-components.md`

**Review by hanjoosten — CHANGES_REQUESTED** [review]
Nice work @MarkCorazon! I have some really minor change requests.

File: docs/documentation.md (hunk 2 above)
Comment (MarkCorazon) [reply docs/documentation.md]: Thats a good point. 23:00 was not my brightest moment i guess.

File: docs/documentation.md (hunk 1 above)
Comment (MarkCorazon) [reply docs/documentation.md]: Check, this is now changed to start with an uppercase

**Review by PatrickR079 — APPROVED** [review]

**Review by LiWeiYeh — APPROVED** [review]

**Pull request merged**